src/workerd/server/tests/python/sdk/worker.py

```diff
@@ -423,7 +428,7 @@ async def response_unit_tests(env):
 
     response_none = Response(None, status=204)
     assert response_none.status == 204
-    assert response_none.body is None
+    assert response_none.body is jsnull
```

Comment:
Hmm. This is the SDK which shouldn't leak `jsnull`, does the sdk need to be updated to transform jsnull into `None`?

Reply:
I guess so.